data/fields/seamark/type.json

```diff
@@ -4,5 +4,333 @@
+            },
+            "coastguard_station": {
+                "title": "Costguard Station",
+                "description": "Watch keeping station at which a watch is kept either continuously, or at certain times only."
+            },
+            "control_point": {
+                "title": "Control Point",
+                "description": "A point on the ground where position (horizontal and vertical) is used as a base for a dependent survey. Also referred to as a control station."
+            },
+            "crane": {
+                "title": "Crane",
+                "description": "A machine used for hoisting and moving heavy objects."
+            },
+            "distance_mark": {
+                "title": "Distance Mark",
+                "description": "A distance mark indicates the distance measured from an origin and consists of either a solid visible structure or a distinct location without special installation."
+            },
+            "dredged_area": {
+                "title": "Dredged Area",
+                "description": "An area of the bottom of a body of water which has been deepened by dredging."
+            },
+            "dumping_ground": {
+                "title": "Dumping Ground",
+                "description": "A sea area where dredged material or other potentially more harmful material, e.g. explosives, chemical waste, is deliberately deposited."
+            },
+            "fairway": {
+                "title": "Fairway",
+                "description": "That part of a river, harbour and so on, where the main navigable channel for vessels of larger size lies."
+            },
+            "ferry_route": {
+                "title": "Ferry Route",
+                "description": "A route in a body of water where a ferry crosses from one shoreline to another."
+            },
+            "gate": {
+                "title": "Gate",
+                "description": "A structure that may be swung, drawn, or lowered to block an entrance or passageway."
+            },
+            "harbour_basin": {
+                "title": "Harbour Basin",
+                "description": "An enclosed area of water surrounded by quay walls constructed to provide means for the transfer of cargoes from and to ships."
+            },
+            "harbour": {
+                "title": "Harbour",
+                "description": "A Harbour installation with a service or commercial operation of public interest. A body of water providing protection for vessels, and providing anchoring and docking facilities."
+            },
+            "hulk": {
+                "title": "Hulk",
+                "description": "The hull of a wrecked or condemned ship, which is moored in a permanent position or grounded."
+            },
+            "landmark": {
+                "title": "Landmark",
+                "description": "A prominent object at a fixed location which can be used in determining a location or a direction."
```

Comment:
how mapper is supposed to know which ones are eligible?

how it works when object has normal object type + this seamark? What iD displays in such case?